modules/recon/contacts-contacts/abc.py

```diff
@@ -0,0 +1,88 @@
+from recon.core.module import BaseModule
+
+from recon.mixins.threads import ThreadingMixin
+
+from bs4 import BeautifulSoup
+
+
+class Module(BaseModule, ThreadingMixin):
+
+    meta = {
+        "name": "Advance Background Check Lookup",
+        "author": "Cam Barts (@cam-barts)",
+        "version": "1.0",
+        "description": "Checks names at advancedbackgroundchecks.com",
+        "dependencies": ["beautifulsoup4"],
+        "comments": ("Inspiration drawn from @xillwillx skiptracer module https://github.com/xillwillx/skiptracer/blob/master/plugins/advance_background_checks.py"),
+        "query": "SELECT DISTINCT first_name, last_name, region FROM contacts WHERE first_name IS NOT NULL AND last_name IS NOT NULL",
+        "options": (
+            ("location", "", True, "location to search (city, state, or zip)"),
+        ),
+    }
+
+    def module_run(self, people):
+        url_names = []
+        for person in people:
+            name = f"{person[0]}-{person[1]}"
+            if person[2]:
+                region = f"{person[2]}".replace(" ", "-")
+            else:
+                region = self.options["location"]
+            url = f"https://www.advancedbackgroundchecks.com/name/{name}_{region}"
+            url_names.append((url, name))
+        self.thread(url_names)
+
+    def module_thread(self, url_name):
+        headers = {"user-agent": "Mozilla/5.0 Chrome/77 Safari/537"}
```

Comment:
Why this? The framework already sets a custom user agent.

Reply:
And there in lies the reason for being able to change the User-Agent in the global configuration.